package.json

```
@@ -7,9 +7,9 @@
     "url": "git+https://github.com/AxaFrance/design-system.git"
   },
   "scripts": {
-    "start": "turbo start start:storybook",
-    "build": "turbo build build:storybook --no-daemon --concurrency 1",
-    "lint": "turbo eslint stylelint prettier tsc --no-daemon",
+    "start": "concurrently \"npm run start --workspace=packages/css\" \"npm run start:storybook --workspaces\"",
```

La commande ne lance qu'un seul des 2 storiebook.

Voici un correctif avec quelque amélioration de couleur et naming dans la console 😉
```suggestion
    "start": "concurrently \"npm:start -w=packages/css\" \"npm:start:storybook -w=packages/css\" \"npm:start:storybook -w=packages/react\" -c auto -n css,css:storybook,react:storybook",
```

---

A un moment il m'avait lancé les deux, par quelle opération du saint esprit je ne sais pas, mais j'avais prévu de retester ce matin sans mes soucis de processeur qui s'envole. Effectivement il n'en lançait qu'une seule.